### ISSUE TITLE
Question: Single editable row in a b-table

### ISSUE THREAD

**Issue by willismonroe** [bootstrap-vue/bootstrap-vue#5322]
I'm trying to figure out the best way to format one row of a b-table with input fields instead of just data.

Right now I'm using templates for each column with v-slot's for each cell and then doing conditional logic inside the template and either showing an input or the data based on a parameter (basically if one element of the row matches a prop).  This seems excessively verbose but I'm not sure another way to do it.

So basically my b-table is full of these, which makes it longer than just constructing a b-table-simple:
```
        <template v-slot:cell(goal)="data">
          <template v-if="data.item.uid === uid">
            <b-form-input v-model="userRow.goal" type="number"
          /></template>
          <template v-else>{{ data.value }}</template>
        </template>
```

**Comment by Hiws** [comment]
You can't really get around the templates, but you can make a more generic template that you can reuse.
It'll depend a bit on the data you need to change, but if it's something simpler, this should work alright for you, or at least cut down a bit of the markup.

I just took a guess at how your current setup is, so it might not be exactly what you want, but hope it at least can help a little.

https://codepen.io/Hiws/pen/oNjybgw

**Comment by willismonroe** [comment]
Your example makes a lot of sense, I didn't think about abstracting the the templates like that, thanks!
